### PR TITLE
Use official denomination for parisian stations and remove hyphens

### PR DESCRIPTION
**Pull request by adhumi**
Remove hyphens + official denomination.

See the following links for each station :

* [FRPNO](https://www.gares-sncf.com/fr/gare/frpno/paris-gare-du-nord)
* [FRPST](https://www.gares-sncf.com/fr/gare/frpst/paris-est)
* [FRPLY](https://www.gares-sncf.com/fr/gare/frply/paris-gare-lyon)
* [FRPBE](https://www.gares-sncf.com/fr/gare/frpbe/paris-bercy-bourgogne-pays-dauvergne)
* [FRPAZ](https://www.gares-sncf.com/fr/gare/frpaz/paris-austerlitz)
* [FRPMO](https://www.gares-sncf.com/fr/gare/frpmo/paris-montparnasse)
* [FRPSL](https://www.gares-sncf.com/fr/gare/frpsl/paris-saint-lazare)

### CODE REVIEW

File: stations.csv [hunk 1]
```diff
@@ -3696,13 +3696,13 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
...
-4921;Paris-Austerlitz;paris-austerlitz;8754700;87547000;48.84091003613604;2.3665666580200195;4916;f;FR;f;Europe/Paris;t;FRPAZ;PAZ;t;;f;8700010;t;;f;;f;8754700;f;;;f;;f;87011;t;;f;t;;;;;;Parigi;Parijs;Paříž;;Párizs;パリ;파리;Paryż;;Париж;;;巴黎
-4922;Paris-Gare-du-Nord;paris-gare-du-nord;8727100;87271007;48.880885;2.354931;4916;f;FR;f;Europe/Paris;t;FRPNO;PNO;t;;f;8700014;f;;f;;f;8727100;f;;;f;;f;8700015;t;;f;t;;;;;;Parigi;Parijs;Paříž;;Párizs;パリ;파리;Paryż;;Париж;;;巴黎
-4923;Paris-Gare-St-Lazare;paris-gare-st-lazare;8738400;87384008;48.876328;2.325338;4916;f;FR;f;Europe/Paris;t;FRPSL;PSL;t;;f;8700015;f;;f;;f;;f;;;f;;f;8700017;t;;f;t;;;;;;Parigi;Parijs;Paříž;;Párizs;パリ;파리;Paryż;;Париж;;;巴黎
-4924;Paris-Gare-de-Lyon;paris-gare-de-lyon;8768600;87686006;48.844839;2.374069;4916;f;FR;f;Europe/Paris;t;FRPLY;PLY;t;PLY;t;8700012;f;;f;;f;8768600;t;;;f;;f;8700013;t;;f;t;;;;;;Parigi;Parijs;Paříž;;Párizs;パリ;파리;Paryż;;Париж;;;巴黎
+4917;Paris Bercy Bourgogne - Pays d’Auvergne;paris-bercy-bourgogne-pays-dauvergne;8768666;87686667;48.839131;2.382887;4916;f;FR;f;Europe/Paris;t;FRPBE;PBY;t;;f;8702455;t;;f;;f;;f;;;f;;f;8700018;t;;f;t;;;;;;Parigi;Parijs;Paříž;;Párizs;パリ;파리;Paryż;;Париж;;;巴黎
```
Comment (mckbrmn) [inline stations.csv]: I'm dislike the dash surrounded by spaces. Could you remove the spaces?

File: stations.csv [hunk 2]
```diff
@@ -3696,13 +3696,13 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
...
+4919;Paris Est;paris-est;8711300;87113001;48.876975;2.35912;4916;f;FR;f;Europe/Paris;t;FRPST;PES;t;PST;f;8700011;t;;f;;f;8711300;f;;;f;;f;8700012;t;;f;t;;;;;;Parigi;Parijs;Paříž;;Párizs;パリ;파리;Paryż;;Париж;;;巴黎
```
Comment (mckbrmn) [inline stations.csv]: Even though it is the official name for the station, I find it confusing and inconsistent. I'd rather have it as _Paris Gare de l’Est_, to stay in line with _Paris Gare du Nord_ (which you can sometimes see written as _Paris Nord_).

Keeping the _Gare de l’Est_ name also stays consistent with what the station's metro stop is called.

File: stations.csv [hunk 3]
```diff
@@ -3696,13 +3696,13 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
...
+4920;Paris Montparnasse;paris-montparnasse;8739100;87391003;48.840488;2.319576;4916;f;FR;f;Europe/Paris;t;FRPMO;PMP;t;PMO;t;8700013;f;;f;;f;;f;;;f;;f;8700014;t;;f;t;;;;;;Parigi;Parijs;Paříž;;Párizs;パリ;파리;Paryż;;Париж;;;巴黎
+4921;Paris Austerlitz;paris-austerlitz;8754700;87547000;48.84091003613604;2.3665666580200195;4916;f;FR;f;Europe/Paris;t;FRPAZ;PAZ;t;;f;8700010;t;;f;;f;8754700;f;;;f;;f;87011;t;;f;t;;;;;;Parigi;Parijs;Paříž;;Párizs;パリ;파리;Paryż;;Париж;;;巴黎
+4922;Paris Gare du Nord;paris-gare-du-nord;8727100;87271007;48.880885;2.354931;4916;f;FR;f;Europe/Paris;t;FRPNO;PNO;t;;f;8700014;f;;f;;f;8727100;f;;;f;;f;8700015;t;;f;t;;;;;;Parigi;Parijs;Paříž;;Párizs;パリ;파리;Paryż;;Париж;;;巴黎
+4923;Paris St-Lazare;paris-st-lazare;8738400;87384008;48.876328;2.325338;4916;f;FR;f;Europe/Paris;t;FRPSL;PSL;t;;f;8700015;f;;f;;f;;f;;;f;;f;8700017;t;;f;t;;;;;;Parigi;Parijs;Paříž;;Párizs;パリ;파리;Paryż;;Париж;;;巴黎
```
Comment (mckbrmn) [inline stations.csv]: If you are moving to the official name using gares-sncf.com as a source, why not expand the `St`, calling the station _Paris Saint-Lazare_?

File: stations.csv (hunk 1 above)
Comment (mckbrmn) [inline stations.csv]: The [press release](http://www.sncf.com/ressources/cp_nouveau_nom_paris_bercy_vdef_3.pdf) announcing the name change doesn't feature the spaces.

It also contains 5 (out of 8 instances!) different variations on the name…

Comment (adhumi) [inline stations.csv]: Mmm, maybe we should stick to the name displayed *in the station* ?

bercy

(I remove the surrounding spaces)

File: stations.csv (hunk 2 above)
Comment (adhumi) [inline stations.csv]: Agree with that. The main point of this pull request is to remove the hyphens, not to introduce confusion.

File: stations.csv (hunk 3 above)
Comment (adhumi) [inline stations.csv]: Well, almost every station with `Saint` is shortened to `St` in this repo. I think it is better to stick with that logic for consistency.

**Comment by adhumi** [comment]
Patched

File: stations.csv [hunk 4]
```diff
@@ -3696,13 +3696,13 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
...
-4922;Paris-Gare-du-Nord;paris-gare-du-nord;8727100;87271007;48.880885;2.354931;4916;f;FR;f;Europe/Paris;t;FRPNO;PNO;t;;f;8700014;f;;f;;f;8727100;f;;;f;;f;8700015;t;;f;t;;;;;;Parigi;Parijs;Paříž;;Párizs;パリ;파리;Paryż;;Париж;;;巴黎
-4923;Paris-Gare-St-Lazare;paris-gare-st-lazare;8738400;87384008;48.876328;2.325338;4916;f;FR;f;Europe/Paris;t;FRPSL;PSL;t;;f;8700015;f;;f;;f;;f;;;f;;f;8700017;t;;f;t;;;;;;Parigi;Parijs;Paříž;;Párizs;パリ;파리;Paryż;;Париж;;;巴黎
-4924;Paris-Gare-de-Lyon;paris-gare-de-lyon;8768600;87686006;48.844839;2.374069;4916;f;FR;f;Europe/Paris;t;FRPLY;PLY;t;PLY;t;8700012;f;;f;;f;8768600;t;;;f;;f;8700013;t;;f;t;;;;;;Parigi;Parijs;Paříž;;Párizs;パリ;파리;Paryż;;Париж;;;巴黎
+4917;Paris Bercy Bourgogne-Pays d’Auvergne;paris-bercy-bourgogne-pays-dauvergne;8768666;87686667;48.839131;2.382887;4916;f;FR;f;Europe/Paris;t;FRPBE;PBY;t;;f;8702455;t;;f;;f;;f;;;f;;f;8700018;t;;f;t;;;;;;Parigi;Parijs;Paříž;;Párizs;パリ;파리;Paryż;;Париж;;;巴黎
+4919;Paris Gare de l'Est;paris-gare-de-lest;8711300;87113001;48.876975;2.35912;4916;f;FR;f;Europe/Paris;t;FRPST;PES;t;PST;f;8700011;t;;f;;f;8711300;f;;;f;;f;8700012;t;;f;t;;;;;;Parigi;Parijs;Paříž;;Párizs;パリ;파리;Paryż;;Париж;;;巴黎
```
Comment (mckbrmn) [inline stations.csv]: You switched from an apostrophe (`’`) to a single quote (`'`). Please restore it.

File: stations.csv (hunk 3 above)
Comment (mckbrmn) [inline stations.csv]: Alright, works for me

File: stations.csv (hunk 4 above)
Comment (adhumi) [inline stations.csv]: Woops, fixed.

**Pull request merged**